includes/admin/settings/register-settings.php

```diff
@@ -1123,11 +1123,11 @@ function edd_radio_callback( $args ) {
 		elseif( isset( $args['std'] ) && $args['std'] == $key && ! isset( $edd_options[ $args['id'] ] ) )
 			$checked = true;
 
-		echo '<input name="edd_settings[' . $args['id'] . ']"" id="edd_settings[' . $args['id'] . '][' . $key . ']" type="radio" value="' . $key . '" ' . checked(true, $checked, false) . '/>&nbsp;';
-		echo '<label for="edd_settings[' . $args['id'] . '][' . $key . ']">' . $option . '</label><br/>';
+		echo '<input name="edd_settings[' . sanitize_key( $args['id'] ) . ']"" id="edd_settings[' . sanitize_key( $args['id'] ) . '][' . esc_attr( $key ) . ']" type="radio" value="' . esc_html( $key ) . '" ' . checked(true, $checked, false) . '/>&nbsp;';
+		echo '<label for="edd_settings[' . sanitize_key( $args['id'] ) . '][' . esc_attr( $key ) . ']">' . esc_html( $option ) . '</label><br/>';
```

Comment:
@pippinsplugins don't have any examples (EDD Core doesn't have many radio fields anyway), its just a thought about of a plausible scenario. 

Haven't been able to test any EDD extension regarding this.